src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java

```diff
@@ -457,7 +457,10 @@ public String initAndValidate(Logger log) throws IllegalArgumentException {
 
     // Initialize the dockerfile location and the build mode
     private void initDockerFileFile(Logger log) {
-        if (dockerFile != null) {
+        if (dockerFile != null && dockerFileDir != null) {
+            dockerFileFile = new File(dockerFileDir, dockerFile);
```

Comment:
Could you please add a check, that `dockerFile` is not an absolute path and throw an error if it is ?